### ISSUE TITLE
Erroneous Master Caution trigger during Fire Warnings

### ISSUE THREAD

**Issue by THEONEANDONLYPLANE** [mSparks43/747-400#643]
**Describe the bug**

The Master Caution warning erroneously triggers during fire tests, and fire events.

# Please provide documentation references:

Place an x to check the boxes:
- [x] Something found during a flight test
- [X] CBT/FCOM reference included (issues are likely to be closed without this)
- [X] checked [current](https://github.com/mSparks43/747-400/blob/master/plugins/xtlua/scripts/B747.68.xt.fms/activepages/version.lua) FMC revision ident 
- [X] Video included (where applicable)
- [ ] Screenshot included (where applicable)

### Bug reports:

The Master Caution warning erroneously triggers during fire tests, and fire events. Please reference the FCOM, the CBT videos. This seems to be a dataref issue, here are the reference footages:

https://www.youtube.com/watch?v=MfoHKdYOCEk

https://www.youtube.com/watch?v=7U8w6B8HnOs

https://www.youtube.com/watch?v=xGbxpq_TM2c

https://www.youtube.com/watch?v=R1l2t0Nj6wM

According to @Matt752 this also happens during Autopilot operations- warnings erroneously trigger.

**Comment by Matt726-S** [comment]
also during the fire test, the fuel control switches should light red and there should be an eicas red message saying fire test in prog, and fire test passed when everything is good, or something like that. also the fire test button tests the cargo fire switches too, but they dont light up in the sparky. https://drive.google.com/file/d/1Ou32g_McdrHZDROJkx_z-wCt3EI4cJNz/view?usp=sharing
please fix this because i believe the aircraft isnt airworthy without the fire test requirements

**Comment by mSparks43** [comment]
confused face.
This clearly shows master caution active
![Screenshot_20210828_214841](https://user-images.githubusercontent.com/6773686/131229428-bbd101ec-54f8-417c-b275-92a7d87fd845.png)

**Comment by THEONEANDONLYPLANE** [comment]
No, Master caution and caution are two different things. Watch the videos I sent please.

**Comment by mSparks43** [comment]
do you mean master warning?

**Comment by THEONEANDONLYPLANE** [comment]
The master caution warning sound, yes. Take a look at this. https://www.youtube.com/watch?v=xGbxpq_TM2c You can see the warning light illuminates, but the aural warning does not play. That's what I've been talking about. Currently, the warning erroneously plays. This is true for other systems as well. As Matt said, the EICAS messages are missing as well.

**Comment by mSparks43** [comment]
There's master caution or master warning. "master caution warning" isn't a thing afaik.
There should be pages in the fcom for this, and it will get caught as we go through the CBT videos
So the way to do this is make one issue for each CBT video such as
https://www.youtube.com/watch?v=MfoHKdYOCEk
Then reference each bit of the video it either doesn't replicate correctly or is missing completely in a separate issue such as this. with caps of the relevent fcom pages.

**Comment by THEONEANDONLYPLANE** [comment]
As far as I'm concerned, it's any fire event. I don't know about anything else, but you get the point... The master caution alarm shouldn't be triggering with the fire test, or the a fire event.

**Comment by mSparks43** [comment]
> As far as I'm concerned, it's any fire event. I don't know about anything else, but you get the point... The master caution alarm shouldn't be triggering with the fire test, or the a fire event. But, I'll take that into account for the future,.

master caution is the orange light that says caution in the picture I posted.
master warning is a red one that says warning.

**Comment by THEONEANDONLYPLANE** [comment]
Apologies. The master warning bell sounds when the a fire is detected, or a fire test is preformed. This should not be happening. The warning light on the CPT and FO's caution indicator should appear, but a bell should not sound.

**Comment by mSparks43** [comment]
> Apologies. The master warning bell sounds when the a fire is detected, or a fire test is preformed. This should not be happening. The warning light on the CPT and FO's caution indicator should appear, but a bell should not sound.

FCOM says it should, in addition to the APU fire horn
![image](https://user-images.githubusercontent.com/6773686/144716082-9f9bf418-8801-468d-a39f-3eb6f8b57547.png)

**Comment by THEONEANDONLYPLANE** [comment]
@mSparks43 Did you watch the four videos I linked? They all **clearly** show that the master caution does not sound. When the FCOM states the 'flight deck warning bell', that's refering to the 'flight deck fire warning bell'. The master caution **light** activates, but the warnings does not. If you watch any of the videos, that is clearly demonstrated.

**Comment by mSparks43** [comment]
the warning siren wants inhibiting if

`laminar/B747/fire/fire_ovht/button_pos`

`sim/cockpit2/annunciators/engine_fire`  

 `sim/operation/failures/rel_apu_fire`

is greater than 0 and the fire bell playing instead
as long as master warning is > 0
